apps/explorer/src/app/routes/index.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { Route, Routes } from 'react-router-dom';
+import { RouteErrorBoundary } from '../components/router-error-boundary';
+
+import { SplashLoader } from '../components/splash-loader';
+import { SplashScreen } from '../components/splash-screen';
+import Home from './home';
+import routerConfig from './router-config';
+
+export interface RouteChildProps {
+  name: string;
+}
+
+export const AppRouter = () => {
+  const splashLoading = (
+    <SplashScreen>
+      <SplashLoader />
+    </SplashScreen>
+  );
+
+  return (
+    <RouteErrorBoundary>
+      <React.Suspense fallback={splashLoading}>
+        <Routes>
+          <>
+            {routerConfig.map(({ path, component: Component, name }) => (
```

Comment:
The latest react router has a new hook `useRoutes` to which you pass a router config, which is what I think allows you to define nested child routes all in one place. Not sure if you'd want to use here but just thought I'd point it out.

I think one of the big benefits of that is that linking when you are in a route becomes relative to that route.